addon/mixins/components/tooltips.js

```diff
@@ -140,9 +145,38 @@ export default Ember.Mixin.create({
     tooltip = renderTooltip(this.get('element'), tooltipOptions);
 
     this.set('tooltip', tooltip);
+
+    /* Bind observer if manual-triggering mode */
+    if (tooltipOptions.event === 'manual') {
+      if (componentWasPassed) {
+        // Keep track of child tooltip component
+        this.set('tooltipChildComponent', component);
+        // Turn 'tooltipOpen' into a computed property, reading from child tooltip component's 'open' option
+        Ember.defineProperty(this, 'tooltipOpen', Ember.computed.reads('tooltipChildComponent.open'));
+      }
+      Ember.addObserver(this, 'tooltipOpen', this, this.tooltipOpenDidChange);
```

Comment:
Maybe `this.addObserver` for code consistency with the `this.removeObserver` call?